### ISSUE TITLE
Can't move items in graph template as desired (cacti 1.1.2)

### ISSUE THREAD

**Issue by gmuellerfr** [Cacti/cacti#564]
Hello,
I have background areas to visualize daytime, nighttime and weekend.
In version 0.8 I was able to move those items on top of the hierarchy.
Now I can't, if the first regular item is an area, too.
![graph_template_traffic](https://cloud.githubusercontent.com/assets/25848039/24994843/dd519024-202c-11e7-9818-1fbed25ada02.png)
It would be nice, if you can change the behaviour as in version 0.8. 
Kind Regards
Guenter

**Comment by cigamit** [comment]
Are you saying that it breaks the graph, or that when you click the move button, it does not move?  If the later is the case, can you please check the sequence numbering in the graph_templates_item table for this template and see if they are correct and let us know.

**Comment by gmuellerfr** [comment]
Hello,
the items numbers are correct, but I can't place all three "No Task" areas on top followed by an plotted area.
If I start with 
![graph_template_traffic1](https://cloud.githubusercontent.com/assets/25848039/25002487/faf62748-204a-11e7-8a5f-9187845a8361.png)
and try to move "Item # 3" down or "Item # 4" up, I end up with
![graph_template_traffic2](https://cloud.githubusercontent.com/assets/25848039/25002536/37f17f08-204b-11e7-97d8-42df3b8aa82f.png)
Maybe it's only a cosmetic issue when GPRINT types are above the correspondent AREA definition.
But I remember, it was possible in version 0.8 and yes, the "graphic items" jumped over the "GPRINT items" in version 0.8, too, when they were moved.

I've played a little bit with this menu. It does not depend on AREA, I can't place the LINE1 as "Item # 4" below "Item # 1-3" of type Data Source "(No Task):". All I can get is:
![graph_template_traffic3](https://cloud.githubusercontent.com/assets/25848039/25002982/75bebcae-204d-11e7-8c2e-c340df97fc3d.png)
If I try to move "Item # 7" down, I end with
![graph_template_traffic4](https://cloud.githubusercontent.com/assets/25848039/25003156/59311b58-204e-11e7-8d78-8751a2cf5d39.png)

Maybe, it has something to do with the automagic moving of the corresponding GPRINT statements I can't remember in version 0.8.

Thank you for listening.
Guenter
